src/main/scala/cognite/spark/v1/DefaultSource.scala

```diff
@@ -102,6 +117,8 @@ class DefaultSource
           inferSchema,
           inferSchemaLimit,
           collectSchemaInferenceMetrics)(sqlContext)
+      case "sequenceRows" =>
```

Comment:
We use lowercase for those, `sequencerows` (just like `3dmodelrevisionnodes` etc.)